code/server/dbscripts/testmatches.sql

```diff
@@ -0,0 +1,5 @@
+insert into user_matches (liker_id, likee_id, likes, createdAt, updatedAt) values (202, 11, 1,  '2016-02-01 03:22:27.833 +00:00', '2016-02-01 03:22:27.833 +00:00');
```

Comment:
This script will likely fail if the user does a DB clean wipe and fill since the IDs will only go up to 200. I think a better solution would be to utilize parameters in the SQL script that we could find and replace when we execute the scripts in database.js. Then we could have an option when we run the server which accepts a parameter (a user id) and fill the user_matches table with matches based off the passed user id and any of the other user ids which are under 200.